src/explore-education-statistics-admin/src/pages/publication/__tests__/PublicationAdoptMethodologyPage.test.tsx

```diff
@@ -0,0 +1,157 @@
+    title: 'Methodology 1',
+    owningPublication: {
+      id: 'publication-2',
+      title: 'Publication 2',
+    },
+  };
+  const testMethodology2: BasicMethodologyVersion = {
+    amendment: false,
+    id: 'methodology-v2',
+    latestInternalReleaseNote: 'this is the release note',
+    methodologyId: 'methodology-2',
+    published: '2021-06-08T00:00:00',
+    slug: 'methodology-slug-2',
+    status: 'Approved',
+    title: 'Methodology 2',
+    owningPublication: {
+      id: 'publication-3',
+      title: 'Publication 3',
+    },
+  };
+  const testMethodology3: BasicMethodologyVersion = {
+    amendment: false,
+    id: 'methodology-v3',
+    latestInternalReleaseNote: 'this is the release note',
+    methodologyId: 'methodology-3',
+    published: '2021-06-08T00:00:00',
+    slug: 'methodology-slug-3',
+    status: 'Approved',
+    title: 'Methodology 3',
+    owningPublication: {
+      id: 'publication-4',
+      title: 'Publication 4',
+    },
+  };
+
+  const testContact: PublicationContactDetails = {
+    id: 'contact-1',
+    contactName: 'John Smith',
+    contactTelNo: '0777777777',
+    teamEmail: 'john.smith@test.com',
+    teamName: 'Team Smith',
+  };
+
+  const testPublication: MyPublication = {
+    id: 'publication-1',
+    title: 'Publication 1',
+    contact: testContact,
+    releases: [],
+    legacyReleases: [],
+    methodologies: [],
+    themeId: 'theme-1',
+    topicId: 'topic-1',
+    permissions: {
+      canAdoptMethodologies: true,
+      canCreateReleases: true,
+      canUpdatePublication: true,
+      canUpdatePublicationTitle: true,
+      canUpdatePublicationSupersededBy: true,
+      canCreateMethodologies: true,
+      canManageExternalMethodology: true,
+    },
+  };
+
+  const testMethodologies: BasicMethodologyVersion[] = [
+    testMethodology1,
+    testMethodology2,
+    testMethodology3,
+  ];
+
+  test('renders the adopt a methodology page correctly', async () => {
+    publicationService.getAdoptableMethodologies.mockResolvedValue(
+      testMethodologies,
+    );
+    renderPage(testPublication);
+
+    expect(screen.getByText('Adopt a methodology')).toBeInTheDocument();
+
+    await waitFor(() =>
+      expect(screen.getByText('Select a methodology')).toBeInTheDocument(),
+    );
+
+    expect(
+      screen.getByLabelText('Search for a methodology'),
+    ).toBeInTheDocument();
+
+    const radios = screen.getAllByRole('radio');
+    expect(radios.length).toBe(3);
+    expect(screen.getByLabelText('Methodology 1')).toHaveAttribute(
+      'value',
+      'methodology-1',
+    );
+    expect(screen.getByLabelText('Methodology 2')).toHaveAttribute(
+      'value',
+      'methodology-2',
+    );
+    expect(screen.getByLabelText('Methodology 3')).toHaveAttribute(
+      'value',
+      'methodology-3',
+    );
+
+    expect(screen.getByRole('button', { name: 'Save' })).toBeInTheDocument();
+    expect(screen.getByRole('button', { name: 'Cancel' })).toBeInTheDocument();
+  });
+});
+
```

Comment:
We're probably missing a few useful cases here for when:
- There are no methodologies
- Successfully submitting the form
- Cancelling the form